<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollegeList</class>
 <widget class="QDialog" name="CollegeList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>381</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>List of College Options</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the Starting College Number: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="beginTrip">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>550</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin Trip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Highlight every college you would like to visit by clicking the name.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: You must click the name of your starting college and specify the number below!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
